src/components/list/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../button';
+import Icon from '../icon';
+
+class List extends React.Component {
+  render() {
+    return (
+      <div className="intellyo-list">
+        { this.props.items.map(this.props.renderItem) }
+        { this.props.isEditing &&
+          <div
+            className="list-new-button-wrap"
+            onClick={ this.props.onAddClick }
+          >
+            <Button
+              icon={ <Icon icon="ion-plus-round" /> }
+              neutral={ true }
+            />
+            <div>{ this.props.newItemDescription }</div>
+          </div>
+        }
+      </div>
+    );
+  }
+}
+
+List.displayName = 'List';
+
+List.propTypes = {
+  items: PropTypes.arrayOf(PropTypes.any),
+  onAddClick: PropTypes.func,
+  newItemDescription: PropTypes.node,
+  renderItem: PropTypes.func.isRequired,
+  isEditing: PropTypes.bool,
+};
+
+List.defaultProps = {
+  newItemDescription: 'Add new item',
+  isEditing: false
```

Comment:
it's not necessary to set a defaultProp for this because if it's not set it's false implicitly. Just saying.

Reply:
✅